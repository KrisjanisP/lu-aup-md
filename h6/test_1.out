ab 2
bc 2
aa 1
cc 1
de 1
ef 1
ff 1
