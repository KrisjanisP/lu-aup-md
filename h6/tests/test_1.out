bc 2
aa 1
ab 1
cc 1
de 1
ef 1
ff 1
